### ISSUE TITLE
Hello, I write the code to train model based on your baseline on Mars, same model and same sample strategy, then test this baseline model using other's open test code , however, the result don't achieve your result in your paper, did you make same experiments to compare the difference?

### ISSUE THREAD

**Issue by flysnowtiger** [jackie840129/STE-NVAN#2]

**Comment by jackie840129** [comment]
Hi,
Can I know which test code you use?
I follow the criterion of this paper : https://zpascal.net/cvpr2018/Li_Diversity_Regularized_Spatiotemporal_CVPR_2018_paper.pdf

**Comment by flysnowtiger** [comment]
thanks  for your reply, and I use this test code, https://github.com/jiyanggao/Video-Person-ReID, maybe you could check your baseline using this.

**Comment by solucky-95** [comment]
Hi,

Thanks for your codes. Great work.
I'm also trying to use the baseline code. I found that the performance of your baseline reported in your paper is almost the state-of-the-art compared with most existing re-id works. Do you know why the performance of your baseline is so good? Is it just because of your evaluation strategy as @snowtiger250 mentioned? Or the math.ceil in the process of RRS?

Best.

**Comment by jackie840129** [comment]
Hi 
Thanks for your question. Because of the page limitation in BMVC, I didn't report why my baseline score is higher than most of the works.
The key point is the FPL. Purely adding a BN, separately training Triplet and ID loss in different places and using the vector after BN for testing will boost the performance.
(I saw this experiment of image-based re-id. https://github.com/L1aoXingyu/reid_baseline)
If without this technique, the baseline performance will be just 85.15% in R1, which is almost same as lots of baseline works reported in their papers.

ps. I forgot to describe this phenomenon in the repo. I will add this in a few days!
Thanks!

**Comment by solucky-95** [comment]
Thanks. Can I ask one more question about the sampling strategy?

Does the RRS strategy bring significant improvement in video-based re-id? Normally, in most existing works, ResNet-50 as a baseline can only reach approximate Rank@1 75%-81%. Rank@1 85% outperforms most state-of-the-arts. Can you kindly recommend another paper in which the baseline achieves around Rank@1 85%? It would be helpful. Thank you very much.

Best.

**Comment by jackie840129** [comment]
I think there are three reasons.
1. Batch hard trip + Cross Entropy, I think no matter using a batch norm or not, jointly using two losses will boost the performance. (Only STA (AAAI19) uses this training strategy, but I think many image-based methods do.)
2. The #frame. If I use six frames, the R1 acc will be 84.5%
3. RRS strategy. I've tried using all frames like [this paper](http://openaccess.thecvf.com/content_cvpr_2018/CameraReady/1036.pdf) and I got similar performance in the paper (83%). But I think this strategy is not efficient.
